### PR TITLE
WT-10274 Improve RTS logs readability

### PR DESCRIPTION
**Pull request by clarissecheah**
- Analyse existing RTS logs for possible improvements
- Improve the readability and understandability of identified RTS logs

### CODE REVIEW

File: src/rollback_to_stable/rts_btree_walk.c [hunk 1]
```diff
@@ -299,7 +299,7 @@ __wt_rts_btree_walk_btree(WT_SESSION_IMPL *session, wt_timestamp_t rollback_time
     btree = S2BT(session);
     conn = S2C(session);
 
-    __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
+    __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_4,
```
Comment (korteland) [inline src/rollback_to_stable/rts_btree_walk.c:302]: This bit needs to be `__wt_verbose_multi_level` (or something along those lines) before this works.

**Review by korteland — COMMENTED** [review]
Pretty much there, one issue that the compiler will hopefully get.

File: src/rollback_to_stable/rts_btree_walk.c (hunk 1 above)
Comment (clarissecheah) [reply src/rollback_to_stable/rts_btree_walk.c]: Yeah, I've just realised the same. That was silly of me :p Fixed here [e582ef9](https://github.com/wiredtiger/wiredtiger/pull/8943/commits/e582ef9943ac58245276835dcd69db9bbb221427)

**Review by korteland — APPROVED** [review]
Changes look good, not sure what the patch build failure is off the top of my head sorry.

**Pull request merged**